app/containers/SignInPage/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withStyles from '@material-ui/core/styles/withStyles';
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+
+import PaperFrame from 'components/PaperFrame';
+import SignInForm from 'containers/SignInForm';
+
+const styles = theme => ({
```

Comment:
Containers shouldn't contain styles. Please create another component or add this style to the `PaperFrame` component.